src/WebServer.Logging/ILogger.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace Restup.WebServer.Logging
+{
+    public interface ILogger
```

Comment:
ah, yeah, how most logger frameworks do this is by bringing it one level up, because having to type a ()=> "message" every time is annoying and it can be useful to know if a certain logger is enabled or not. 

so it would be like:
```
if (_logger.IsInfoEnabled()) {
   var calculatedAvgResponse = CalculateAvgResponse();
   _logger.Info("Statistics, average response time: {calculatedAvgResponse}");
}
```